### PR TITLE
added gke_iowa

### PR DESCRIPTION
**Pull request by swetaraj-19**

### CODE REVIEW

File: environments/prod/gke_resources.auto.tfvars [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+1. Cluster Creation: Builds the GKE clusters (infrastructure foundation).
+2. Provider & Namespace: Configures the Kubernetes provider and creates namespaces for resource organization.
+3. Controller Deployment: Deploys core controllers to manage the system.
+4. Secrets Creation: Securely stores sensitive configuration data.
+5. Resource Deployment: Deploys the actual application components (runner sets, agent pools, etc.).
+*/
+
+gke_resources = {
+  clusters = {
+    gke_devops_ss = {
```
Comment (cgsyam) [inline environments/prod/gke_resources.auto.tfvars]: Change the `devops_ss` name

File: environments/prod/main.tf [hunk 2]
```diff
@@ -2,4 +2,7 @@ module "main" {
   source                    = "../../root"
   artifact_registry_configs = var.artifact_registry_configs
   iam_configs               = var.iam_configs
-}
\ No newline at end of file
+  gke_configs               = var.gke_configs
```
Comment (cgsyam) [inline environments/prod/main.tf:5]: Remove the empty new line.

File: environments/prod/gke_resources.auto.tfvars [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+            }
+            spec = {
+              helm = {
+                auth                   = "gcpserviceaccount"
+                gcpServiceAccountEmail = "sa-gke-devops-sr-1@prj-appcode-gke-prod-da16.iam.gserviceaccount.com"
+                chart                  = "gha-runner-scale-set-controller"
+                includeCRDs            = true
+                namespace              = "ghe-runner-scale-set-controller-1"
+                releaseName            = "ghe-runner-scale-set-controller-1"
+                repo                   = "oci://us-central1-docker.pkg.dev/prj-appcode-gke-prod-da16/artifreg-devops-prod"
```
Comment (cgsyam) [inline environments/prod/gke_resources.auto.tfvars]: Change the project

File: environments/prod/variables.tf [hunk 4]
```diff
@@ -22,3 +22,81 @@ variable "iam_configs" {
...
+        all               = []
+        default-node-pool = []
+      })
+    }))
+  })
+}
+
+variable "gke_resources" {
+  description = "GKE resources"
+  # type = object({
```
Comment (cgsyam) [inline environments/prod/variables.tf]: Why this is commented?

Comment (swetaraj-19) [reply environments/prod/variables.tf]: we added the comment - type mismatch error

Comment (cgsyam) [reply environments/prod/variables.tf]: Ah ok. Then let's remove commented code. Keep only the necessary.

**Pull request merged**